isis/tests/FunctionalTestsJigsaw.cpp

```diff
@@ -143,4 +145,97 @@ TEST_F(ObservationPair, FunctionalTestJigsawCamSolveAll) {
   // DEC(t3) final
   EXPECT_NEAR(elems.at(56).toDouble(), 0.365717165,          0.00001); 
   
-}
\ No newline at end of file
+}
+
+TEST_F(ObservationPair, FunctionalTestJigsawErrorNoSolve) {
+  // delete to remove old camera for when cam is updated
```

Comment:
I am confused by this comment.

Reply:
I can delete, left over from a different test where I needed to delete the testCubes before the jigsaw call